.travis.yml

```diff
@@ -8,120 +8,147 @@ stages:
-              - "target"
-              - "project/target"
-              - "project/project/target"
-              - "project/project/project/target"
-              - "dist"
-              - "build"
-
-      - stage: test
-        name: tests (junit, scalacheck, et al)
-        if: type = pull_request OR repo != scala/scala
-        workspaces:
-          use: bootstrapped
-        script:
-          - set -e
-          - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
-          - sbt -Dstarr.version=$STARR setupValidateTest Test/compile testAll1
-
-      - name: tests (partest)
-        if: type = pull_request OR repo != scala/scala
-        workspaces:
-          use: bootstrapped
-        script:
-          - set -e
-          - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
-          - sbt -Dstarr.version=$STARR setupValidateTest testAll2
-
-      - name: ensure standard library is buildable by Scala 3
-        if: type = pull_request OR repo != scala/scala
-        workspaces:
-          use: bootstrapped
-        script:
-          - set -e
-          - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
-          - sbt -Dscala.build.compileWithDotty=true library/compile
-
-      - stage: test
-        name: build benchmarks (bootstrapped)
-        if: type = pull_request OR repo != scala/scala
-        workspaces:
-          use: bootstrapped
-        script:
-          - set -e
-          - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
-          - sbt bench/Jmh/compile
-
-      - stage: build
-        name: language spec (Jekyll)
-        # wkhtmltopdf requires libssl1.1, which we can't install on xenial
-        dist: bionic
-        language: ruby
-        install:
-          - ruby -v
-          - gem install bundler
-          - bundler --version
-          - bundle install
-          # cribbed from https://github.com/SebastiaanKlippert/go-wkhtmltopdf/blob/master/.travis.yml
-          - sudo apt-get update
-          - sudo apt-get install -y build-essential xorg xfonts-75dpi libpng16-16 libssl1.1
-          - wget --quiet "https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.bionic_amd64.deb"
-          - sudo dpkg -i "wkhtmltox_0.12.6-1.bionic_amd64.deb"
-          - rm "wkhtmltox_0.12.6-1.bionic_amd64.deb"
-        script:
-          - set -e
-          - (cd admin && ./init.sh)
-          - bundle exec jekyll build -s spec/ -d build/spec
-          - export JEKYLL_ENV=spec-pdf
-          - bundle exec jekyll build -s spec/ -d build/spec-pdf
-          - ./scripts/generate-spec-pdf.sh
-        after_success:
-          - ./scripts/travis-publish-spec.sh
+  include:
+    - stage: build
+      if: type != pull_request AND repo = scala/scala
+      name: publish to scala-integration or sonatype
```

Comment:
What does it use to decide to publish to sonatype?  And I think the bootstrap part is useful to know, reasons for removing?

Reply:
if `SCALA_VER_BASE` is defined publishing goes to sonatype
https://github.com/scala/scala/blob/52cdad95982e0bf05d00a0caaaba7d6d2b490a2d/scripts/bootstrap_fun#L35-L48

i'll put back the `bootstrap`